index.md

```diff
@@ -164,27 +159,11 @@ WorkflowHub intends to harvest workflows from existing workflow repositories, in
  
 </details>
 
-## Citation
-
-If you would like to reference the WorkflowHub from academic work, please cite:
-
-Carole Goble; Stian Soiland-Reyes; Finn Bacall; Stuart Owen; Alan Williams; Ignacio Eguinoa; Bert Droesbeke; Simone Leo; Luca Pireddu; Laura Rodríguez-Navas; José Mª Fernández; Salvador Capella-Gutierrez; Hervé Ménager; Björn Grüning; Beatriz Serrano-Solano; Philip Ewels; Frederik Coppens (2021):  
-**Implementing FAIR Digital Objects in the EOSC-Life Workflow Collaboratory**.  
-_Zenodo_  
-<https://zenodo.org/record/4605654>
-
-See also [publication and outreach](outreach/) for more specific work.
```

Comment:
I think we'd probably still want this on the front page, or at least linked from somewhere prominent

Reply:
Yes, this was requested to be added on front page in #23 even though we already had it under Publications, it was not found.